src/main/scala/codacy/tailor/Tailor.scala

```diff
@@ -96,8 +88,7 @@ object Tailor extends Tool {
     }.flatten
   }
 
-  private def parseToolResult(path: Path,
-                              output: List[String]): Try[List[Result]] = {
+  private def parseToolResult(path: Path, output: List[String]): Try[List[Result]] = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [parseToolResult should have a smaller scope by using private[this]](https://app.codacy.com/gh/codacy/codacy-tailor/pullRequest?prid=4816796)